### PR TITLE
Getting Started Page updates

### PR DESCRIPTION
**Pull request by mtreadwa**
-added a sign up link for email notifications
-updated bulleted list of Features
-added a list of datasets
-added a list of analysis workflows
-added short section about creating an account (I know we want more detail here for external users in the future)
-added a citation resource table

In the data request page, added a link to the Creating an Account section.

### CODE REVIEW

**Comment by claymcleod** [comment]
@mtreadwa I have added some changes and some `!!! todo` blocks. My last thought is that we should condense the "Forms" and "Glossary" sections down to one page. My feeling the table of contents header is too crowded at the moment. This will at least get us closer while we work on the bigger overhaul to the content hierarchy.

**Comment by claymcleod** [comment]
@adthrasher would you mind also taking a pass through these changes and seeing what you think?

**Comment by mtreadwa** [comment]
@claymcleod I moved the Glossary items into the FAQ page. This seems like an okay solution to crowded left side bar for now. It's a little awkward for the data access units, but I know each unit like PCGP will have it's own page in the future. I'm not sure where is a good place for Forms to go. Thoughts?

File: docs/faq.md [hunk 1]
```diff
@@ -1,6 +1,92 @@
...
+#### What is a Data Access Unit?
+
+A St. Jude Cloud **Data Access Unit (DAU)** is a grouping of data that typically corresponds to a project, study, or dataset generated at the same time at the same institution. Each DAU has its own governing body of researchers, the [Data Access Committee](#what-is-a-data-access-committee), who preside over the data and who may grant or deny access. We currently have the 5 DAUs detailed below.
+
+##### Pediatric Cancer Genome Project (PCGP)
+PCGP is a paired-tumor normal dataset focused on discovering the genetic origins of pediatric cancer.
+The Pediatric Cancer Genome Project is a collaboration between St. Jude Children's Research Hospital and the McDonnell Genome Institute at Washington University School of Medicine that sequenced the genomes of over 600 pediatric cancer patients. 
+
+##### St. Jude Lifetime (SJLIFE)
+SJLIFE is a germline-only dataset focused on studying the long-term adverse outcomes associated with cancer and cancer-related therapy.
```
Comment (adthrasher) [inline docs/faq.md]: I think this line should be removed. It is all stated more clearly in the next section.

File: docs/faq.md [hunk 2]
```diff
@@ -1,6 +1,92 @@
...
+St. Jude Lifetime (SJLIFE) is a longevity study from St. Jude Children's Research Hospital that aims to identify all inherited genome sequence and structural variants influencing the development of childhood cancer and occurrence of long-term adverse outcomes associated with cancer and cancer-related therapy. This cohort contains unpaired germline samples and does not contain tumor samples. 
+
+##### Clinical Genomics (Clinical Pilot and G4K)
+Clinical Genomics is a paired tumor-normal dataset focused on identifying variants that influence the development and behavior of childhood tumors.
```
Comment (adthrasher) [inline docs/faq.md]: Same as above.

File: docs/faq.md [hunk 3]
```diff
@@ -1,6 +1,92 @@
...
+Clinical Genomics is a cohort from St. Jude Children's Research Hospital, comprised of two studies: Clinical Pilot and Genomes4Kids. Clinical Pilot is a smaller, pilot study generated to asses the validity and accuracy of moving forward with the G4K study. These studies aim to identify all inherited and tumor-acquired (somatic) genome sequence and structural variants influencing the development and behavior of childhood tumors. 
+
+##### Sickle Cell Genome Project (SGP)
+SGP is a germline-only dataset of Sickle Cell Disease (SCD) patients from birth to young adulthood.
```
Comment (adthrasher) [inline docs/faq.md]: same

File: docs/faq.md [hunk 4]
```diff
@@ -1,6 +1,92 @@
...
+The Sickle Cell Genome Project (SGP) is a collaboration between St. Jude Children’s Research Hospital and Baylor College of Medicine focused on identifying genetic modifiers that contribute to various health complications in SCD patients. Additional objectives include, but are not limited to, developing accurate methods to characterize germline structural variants in highly homologous globin locus and blood typing.
+
+##### Childhood Cancer Survivor Study (CCSS)
+CCSS is a germline-only dataset consisting of whole genome sequencing of childhood cancer survivors.
```
Comment (adthrasher) [inline docs/faq.md]: same

File: docs/faq.md [hunk 5]
```diff
@@ -1,6 +1,92 @@
...
+    three data access committees for approval. Since each DAC is made up of
+    different individuals using different criteria for evaluation, you may or
+    may not be approved for access to all of the files. 
+
+#### What is a Data Access Unit?
+
+A St. Jude Cloud **Data Access Unit (DAU)** is a grouping of data that typically corresponds to a project, study, or dataset generated at the same time at the same institution. Each DAU has its own governing body of researchers, the [Data Access Committee](#what-is-a-data-access-committee), who preside over the data and who may grant or deny access. We currently have the 5 DAUs detailed below.
+
+##### Pediatric Cancer Genome Project (PCGP)
+PCGP is a paired-tumor normal dataset focused on discovering the genetic origins of pediatric cancer.
```
Comment (adthrasher) [inline docs/faq.md]: same as below.

File: docs/faq.md [hunk 6]
```diff
@@ -1,6 +1,92 @@
...
+
+!!! warning "Potential Bacterial Contamination"
+
+    Samples for the Childhood Cancer Survivorship Study were collected by sending out Buccal swab kits to enrolled participants and having them complete the kits at home. This mechanism of collecting saliva and buccal cells for sequencing is highly desirable because of its uninvasive nature and ease of execution. However, collection of samples in this manner also has higher probability of contamination from external sources (as compared to, say, samples collected using blood). We have observed some samples in this cohort which suffer from bacterial contamination. To address this issue, we have taken the following steps:
```
Comment (adthrasher) [inline docs/faq.md]: uninvasive -> non-invasive

**Comment by claymcleod** [comment]
@mtreadwa can we merge this in today (Wednesday)? It would be best if so, there are some things I want to refer an external user to using the updated docs.

**Pull request merged**